NDTensors/test/device_list.jl

```diff
@@ -0,0 +1,21 @@
+function devices_list(test_args)
+  devs = Vector{Function}(undef, 0)
+  if isempty(test_args) || "base" in test_args
+    push!(devs, NDTensors.cpu)
+  end
+
+  if "cuda" in test_args || "all" in test_args
+    CUDA.allowscalar()
+    if CUDA.functional()
+      push!(devs, NDTensors.cu)
+      #include(joinpath(pkgdir(NDTensors), "ext", "examples", "NDTensorCUDA.jl"))
+    end
+  end
+
+  if "metal" in test_args || "all" in test_args
+    push!(devs, NDTensors.mtl)
+    Metal.allowscalar()
+    include(joinpath(pkgdir(NDTensors), "ext", "examples", "NDTensorMetal.jl"))
```

Comment:
Should this be removed?

Reply:
Oops sorry, I misread this, I thought it was loading the extension package but I see it is loading the example.